### PR TITLE
Add remodels of the Photovoltaic outfits

### PR DESCRIPTION
**Pull request by GefullteTaubenbrust2**
## Summary
Adds reworked sprites for the tiny and small photovoltaic outfits.

comparison

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/81
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/302

### CODE REVIEW

File: copyright [hunk 1]
```diff
@@ -1426,10 +1420,12 @@ Files:
  images/outfit/nuke*
  images/outfit/plasma?core*
  images/outfit/red?sun*
+ images/outfit/small?photovoltaic*
```
Comment (beccabunny) [inline copyright:1423]: These should be on a separate entry, they aren't only derived from MZ

Comment (GefullteTaubenbrust2) [reply copyright:1423]: Oh I see. Right, I'll fix it right away.

File: copyright [hunk 2]
```diff
@@ -1459,6 +1453,13 @@ Files:
 Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
 License: CC-BY-SA-4.0
 
+Files:
+ images/outfit/small?photovoltaic*
+ images/outfit/tiny?photovoltaic*
+Copyright: Gefüllte Taubenbrust <jeaminer23@gmail.com>
+License: CC-BY-SA-4.0
```
Comment (beccabunny) [inline copyright:1460]: Old license is 3.0, are we allowed to make the new one 4.0?

Comment (GefullteTaubenbrust2) [reply copyright:1460]: Idk honestly.

**Comment by Hurleveur** [comment]
I'll be honest I kinda like the old models more for the nice looking space reflections.
The new ones may be more realistic because its more resilient though?

You should probably not hold this off just for me, art often is subjective.

**Comment by GefullteTaubenbrust2** [comment]
> I'll be honest I kinda like the old models more for the nice looking space reflections. The new ones may be more realistic because its more resilient though?
> 
> You should probably not hold this off just for me, art often is subjective.

I never really thought about them as star reflections, mostly because it doesn't make a whole lot of sense that they would be visible when the panel is fully in sunlight. Anyway, I get that it's quite different from the originals and if that doesn't fly with people i can change it, but we'll see :)

File: copyright (hunk 2 above)
Comment (tibetiroka) [reply copyright:1460]: If I'm reading [this](https://wiki.creativecommons.org/wiki/4.0_upgrade_guidelines) correctly then you can use 4.0 in a context where 3.0 was used previously. Not sure what the benefits are though.

**Comment by Zitchas** [comment]
I'm not sure about the thin grey bars inside the panel, kind of prefer the simple crease-in-the-material of the older images for that part. Other than that, though, I think these look really good. The frame in particular is a nice improvement.

**Comment by GefullteTaubenbrust2** [comment]
> I'm not sure about the thin grey bars inside the panel, kind of prefer the simple crease-in-the-material of the older images for that part. Other than that, though, I think these look really good. The frame in particular is a nice improvement.

Funny that because that was what I originally had (obviously the older version was different in other ways):
small_photovoltaic

**Comment by Pointedstick** [comment]
^^ this is the way

**Review by Zitchas — APPROVED** [review]
Very nice. Looks good to me!

**Pull request merged**